app/assets/javascripts/components/story/CollapsedStory/CollapsedStoryEstimate.js

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const estimateRule = (estimate) => estimate > 0 ? estimate : '-';
+const estimateRule = (estimate) => estimate || '-';
 
 const CollapsedStoryEstimate = ({ estimate }) => (
   <span className='Story__estimated-value'>{estimateRule(estimate)}</span>
 );
 
 CollapsedStoryEstimate.propTypes = {
-  estimate: PropTypes.number,
+  estimate: PropTypes.any,
```

Comment:
You could use it like this to accept both (empty) string and numbers.

```suggestion
  estimate: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
```